tensorflow/lite/delegates/gpu/common/shape.h

```diff
@@ -577,7 +579,7 @@ auto DispatchByLayout(Layout type, F f)
       return f.template operator()<Layout::SCALAR>();
     case Layout::BHWC:
       return f.template operator()<Layout::BHWC>();
-    case Layout::UNKNOWN:
+    default:
```

Comment:
i'm not in favor of default in this case.  if we add a new enum value, the compiler should catch it.  if you add a default, the compiler won't.  what is the warning you're seeing?